### PR TITLE
Lucas/clean up self hosted pricing page 1769

### PR DESCRIPTION
**Pull request by lucasvaltl**
## Description
This change aims to clarify our self-hosted licensing situation by merging the `starter` and `community` licenses or rather plans into one - which is how it is actually implemented. It is related to [this](https://www.notion.so/gitpod/Polished-Self-Hosted-Licensing-User-Flow-b3028667f0c24b86b89753a67b1c3fa7#a0819128643b45e0bcece95c2d93cbde) (internal) effort. 

This is just a first version, it still requires design work to make this pretty. However this is non-trivial (see internal discussion [here](https://gitpod.slack.com/archives/C01KGM9ETU6/p1650876100024629?thread_ts=1650874593.996869&cid=C01KGM9ETU6))

## Related Issue(s)
- Very basic first implementation of #1769 
- Related to broader work on self-hosted docs, see #1942

## How to test
- Open workspace, open preview on port 3000 and navigate to the pricing page, then switch to `self-hosted`

## Release Notes

```release-note
NONE
```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1940"></a>

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1940"></a>

### CODE REVIEW

File: src/lib/contents/self-hosted.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const pricingPlans: Pricing[] = [
     features: [
       "Starts after the 11th user",
       "Everything from Starter",
+      "Air gapped environments",
       "Standard support",
     ],
     btnText: "Register and Install now",
```
Comment (atduarte) [inline src/lib/contents/self-hosted.ts:51]: _Just fyi as you didn't change this._

This text can overflow the button 
image

**Review by atduarte — APPROVED** [review]
LGTM

File: src/lib/contents/self-hosted.ts (hunk 1 above)
Comment (lucasvaltl) [reply src/lib/contents/self-hosted.ts:51]: Must have been there before, as you said. I'll see if I can mitigate it somehow or raise a ticket. Thanks for noticing!

Comment (lucasvaltl) [reply src/lib/contents/self-hosted.ts:51]: Ticket it is :) https://github.com/gitpod-io/website/issues/1964

**Pull request merged**